src/auth-service/.env.example

```diff
@@ -0,0 +1,123 @@
+AIRQO_WEBSITE=https://airqo.net/
+AIRQO_YOUTUBE=
+DB_NAME_STAGING=
```

Comment:
Introduced the template .env file for open source contributions and new codebase setup